src/client/index.ts

```diff
@@ -198,6 +200,9 @@ class Client extends EventEmitter {
   // number. Defaults to '2'.
   public readonly maxFeeXRP: string
 
+  // eslint-disable-next-line @typescript-eslint/no-invalid-this -- this is expected
+  public autofill = partial(autofill, this)
```

Comment:
can you add a TODO to use `partial` for the other methods as well? (you don't need to do it now, especially because we're going to delete a bunch of these methods anyways)

Reply:
Done.